bundles/org.openhab.binding.luftdateninfo/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="luftdateninfo"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="particulate">
+		<label>Particulate Sensor</label>
+		<description>Sensor to measure Particulate Matter (PM)</description>
+
+		<channels>
+			<channel id="pm25" typeId="pm25-channel"/>
+			<channel id="pm100" typeId="pm100-channel"/>
+		</channels>
+
+		<config-description>
+			<parameter name="sensorid" type="integer" required="true">
+				<label>Sensor ID</label>
+				<description>Sensor ID</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<thing-type id="conditions">
+		<label>Condition Sensor</label>
+		<description>Sensor to measure Temperature and Humidity conditions</description>
+
+		<channels>
+			<channel id="temperature" typeId="temp-channel"/>
+			<channel id="humidity" typeId="hum-channel"/>
+			<channel id="pressure" typeId="pressure-channel"/>
+			<channel id="pressure-sea" typeId="pressure-sea-channel"/>
+		</channels>
+
+		<config-description>
+			<parameter name="sensorid" type="integer" required="true">
+				<label>Sensor ID</label>
+				<description>Sensor ID</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<thing-type id="noise">
+		<label>Noise Sensor</label>
+		<description>Sensor to measure noise on location</description>
+
+		<channels>
+			<channel id="noise-eq" typeId="noise-eq-channel"/>
+			<channel id="noise-min" typeId="noise-min-channel"/>
+			<channel id="noise-max" typeId="noise-max-channel"/>
+		</channels>
+
+		<config-description>
+			<parameter name="sensorid" type="integer" required="true">
+				<label>Sensor ID</label>
+				<description>Sensor ID</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="pm25-channel">
+		<item-type>Number</item-type>
+		<label>PM 2.5:Density</label>
```

Comment:
Same for the others.
```suggestion
		<item-type>Number:Density</item-type>
		<label>PM 2.5</label>
```

Reply:
Oh no, my fault! Sure, moved QuantityTypes from "label" to "item-type" at all locations